app/src/main/java/com/amaze/filemanager/filesystem/compressed/extractcontents/Extractor.java

```diff
@@ -97,6 +97,9 @@ public void extractEverything() throws IOException {
     protected String fixEntryName(String entryName){
         if(entryName.indexOf('\\') > 0) {
             return entryName.replaceAll("\\\\", CompressedHelper.SEPARATOR);
+        } if(entryName.indexOf(CompressedHelper.SEPARATOR_CHAR) == 0) {
```

Comment:
You mean `startsWith()`?

Also confusing `if` positioning, looks like an `else if`.